src/praisonai/praisonai/cli/main.py

```
@@ -1997,14 +1997,19 @@ def parse_args(self):
                     print("\npip install praisonaiagents  # native PraisonAI")
                     print("pip install \"praisonai-frameworks[crewai]\"  # CrewAI")
                     print("pip install \"praisonai-frameworks[autogen]\"  # AutoGen")
-                    print("pip install \"praisonai-frameworks[openai-agents]\"  # OpenAI Agents SDK\n")
+                    print("pip install \"praisonai-frameworks[openai-agents]\"  # OpenAI Agents SDK")
+                    print("pip install \"praisonai-frameworks[agno]\"  # Agno")
+                    print("pip install \"praisonai-frameworks[google-adk]\"  # Google ADK\n")
                     sys.exit(1)
             except ImportError:
                 if not CREWAI_AVAILABLE and not AUTOGEN_AVAILABLE and not PRAISONAI_AVAILABLE:
```

Ruff (0.15.20)

[error] Undefined name `CREWAI_AVAILABLE`

(F821)